packages/tx/test/eip4844.spec.ts

```diff
@@ -12,7 +13,13 @@ const isBrowser = new Function('try {return this===window;}catch(e){ return fals
 const pk = randomBytes(32)
 initKZG(kzg)
 
-tape('EIP4844 constructor tests - valid scenarios', (t) => {
+const gethGenesis = require('../../block/test/testdata/post-merge-hardfork.json')
```

Comment:
Ah, we really should stop with this habit referencing test data in other packages, this *will* just lead to some hazzle like "one is correcting the test json file while working on the block package and then something breaks on CI in tx after pushing. 🤔

Don't want to make any huzzle here, just saying.

At the end it's really not such a big deal to add a `shared-test-data` folder as described in #2304, maybe we just want to start on this super short term (and independent from other VM/EVM test structure changing work to not mix things unnecessarily) and just start a new folder and put the first file in and that's it (+ maybe mention somewhere in the docs).

All not for this PR, just saying since it's on my mind. 🙂

Reply:
Totally agree.  This was just a hack to get this test up and running :grimacing: